command/issues/list/testdata/test.sarif

```diff
@@ -0,0 +1,100 @@
+{
+  "version": "2.1.0",
+  "$schema": "https://json.schemastore.org/sarif-2.1.0-rtm.5.json",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "informationUri": "https://deepsource.io/directory/analyzers/go",
+          "name": "DeepSource Go Analyzer",
+          "rules": [
+            {
+              "id": "RVV-B0013",
+              "name": "Unused method receiver detected",
+              "shortDescription": null,
+              "fullDescription": {
+                "text": ""
+              },
+              "helpUri": "https://deepsource.io/directory/analyzers/go/issues/RVV-B0013",
+              "properties": {
+                "category": "",
+                "recommended": ""
+              }
+            }
+          ]
+        }
+      },
+      "results": [
+        {
+          "ruleId": "RVV-B0013",
+          "ruleIndex": 0,
+          "kind": "fail",
+          "level": "error",
+          "message": {
+            "text": "Unused method receiver detected"
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "deepsource/transformers/queries/get_transformers.go"
+                },
+                "region": {
+                  "startLine": 34,
+                  "endLine": 34
+                }
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "tool": {
+        "driver": {
+          "informationUri": "https://deepsource.io/directory/analyzers/go",
+          "name": "DeepSource Go Analyzer",
```

Comment:
@burntcarrot Shouldn't there be a single entry for `Go` analyzer (the above one)? Ideally that should contain all the issues reported by the Go analyzer. 🤔

Reply:
Fixing it.